libs/designsystem/src/lib/components/dropdown/dropdown.component.scss

```diff
@@ -114,3 +114,17 @@ kirby-card {
     }
   }
 }
+
+:host-context(kirby-calendar) {
+  button {
+    border: none; // TODO: Fix glitch when opening dropdown - maybe border-color: transparent?
```

Comment:
Please fix todos...

Reply:
It looks like the glitch occurs because of css applied before we reach the :host-context css. I'm referring to this css `  @include focus() {
    > button,
    > button.attention-level3 {
      border-color: $focus-ring-color;
    }
  }`, which can be found on line 33 - 46 in dropdown.component.scss. I have verified that omitting this section of css, will stop the button from glitching. What should I do here?

Reply:
This has been resolved by adding border-color: transparent and removing border: none from :host-context(kirby-calendar)